### PR TITLE
feat: single instance page

### PR DESCRIPTION
**Pull request by ducanhpham0312**
### Linked issue

Closes ducanhpham0312/zeevision-private#77

### Description

- Implement `/instancee/:id` page with instance content and variables table
- Add corresponding navigation link to the `/instances`, `processes`, `processes/id` pages.
- Add new allowed type `ReactNode` to table content as we need to use `NavLink`.
- Add white background color to Table

### Testing

- Run test bench
- Run zeevision. Remember to `docker compose down -v` before `docker compose up --build`
- Login to pgadmin, create a server and copy-paste this query: https://codefile.io/f/7okeTlDomd
- Open the frontend `localhost:8080/processes`. Clicking an instance key should open the `instance/:id` page with the variables table
- Open `instances`/`process/:id`  page, clicking on the instance key should behave the same
- End time field is included in the instance vertical table.

### CODE REVIEW

**Comment by Anh-Duy-Tran** [comment]
Hi there, if you're or planning to implement the tabs for the tables in the Instance page then I would suggest to grab this as a starting point, saving some implementation time
https://ui.shadcn.com/docs/components/tabs

File: frontend/src/pages/InstancesPage.tsx [hunk 1]
```diff
@@ -23,7 +23,14 @@ export default function InstancesPage() {
                 status,
                 version,
                 startTime,
-              }) => [instanceKey, bpmnProcessId, status, version, startTime],
+              }) => [
+                <NavLink to={instanceKey.toString()}>{instanceKey}</NavLink>,
+                instanceKey,
```
Comment (Anh-Duy-Tran) [inline frontend/src/pages/InstancesPage.tsx:28]: This should be removed, instanceKey is being duplicated.

**Review by Anh-Duy-Tran — APPROVED** [review]
This mostly functions and looks as expected

File: frontend/src/hooks/useQuerySingleInstance.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  instance: Instance;
+}
+
+const SINGLE_INSTANCE_QUERY = (id: string) => gql`
+  query SingleInstance {
+    instance(instanceKey: ${id}) {
+      instanceKey
+      process {
+        bpmnProcessId
+        bpmnResource
```
Comment (kauppie) [inline frontend/src/hooks/useQuerySingleInstance.ts:14]: `bpmnResource` shouldn't be polled.

**Review by kauppie — APPROVED** [review]

File: frontend/src/pages/SingleInstancePage.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          <ResponsiveBpmnViewer
+            navigated
+            className="h-full flex-grow"
+            bpmnString={bpmnResource}
+          />
+        </div>
+      </ResizableContainer>
+      <Table
+        orientation="horizontal"
+        header={["Variable Name", "Variable value", "Time"]}
```
Comment (kauppie) [inline frontend/src/pages/SingleInstancePage.tsx:65]: `Variable value` -> `Variable Value`

**Pull request merged**